types/cesium/index.d.ts

```diff
@@ -2134,8 +2134,9 @@ declare namespace Cesium {
         height: Property;
         scaleByDistance: Property;
         translucencyByDistance: Property;
-        disableDepthTestDistance: Property;
+        disableDepthTestDistance: Property | number;
         pixelOffsetScaleByDistance: Property;
+        heightReference?: Property;
```

Comment:
It's correct that this property is optional (and others as well), though the documentation does not say it is. It may be worth moving all of them to be optional to be more correct with the source.